Assets/HoloToolkit/Utilities/Shaders/BlinnPhongConfigurable.shader

```diff
@@ -1,6 +1,6 @@
 //Very fast shader that uses the Unity lighting model
 //Compiles down to only performing the operations you're actually using
-//Uses material property drawers rather than a custom editor for ease of maintainance
+//Uses material property drawers rather than a custom editor for ease of maintenance
```

Comment:
Please add another commit which will fix converting Tabs to spaces in all the shaders.